user_sync/rules.py

```diff
@@ -403,6 +404,15 @@ def read_desired_user_groups(self, mappings, directory_connector):
                 exec(options['after_mapping_hook'], self.after_mapping_hook_scope)
                 self.log_after_mapping_hook_scope(after_call=True)
 
+                # skip user if hook specifies true
+                skip_user = self.after_mapping_hook_scope['skip_user']
+                if type(skip_user) is bool and skip_user:
```

Comment:
The "Pythonic" way to say this (which is also much more efficient) is:
`if skip_user is True:`

Reply:
Thanks for that! My brain is always in Java, or I have no excuse for missing that one :)